File: 3/3_B.py

s = str(input())
t = str(input())

atcoder = list("atcoder")
for i in range(len(s)):
    if s[i] == "@" and t[i] in atcoder:
        pass
    elif t[i] == "@" and s[i] in atcoder:
        pass
    elif t[i] == s[i]:
        pass
    else:
        print("You will lose")
        exit()

print("You can win")

File: 3/3_B.cpp
//encoding:utf-8

#include <iostream>
#include<string>
#include<functional>
using namespace std;

int main(int argc,char** argv){
  string s,t;
  cin>>s>>t;
  bool ans=true;

  for(int i;i<s.size();i++){
    if(s[i]==t[i]) continue;
    if(t[i]=='@') swap(s[i],t[i]);
    if(s[i]=='@'&&string("atcoder").find(t[i]) != string::npos)continue;
    else{
      ans=false;
    }
  }
  if(ans){
    cout<<"You can win"<<"\n";
  }
  else{
    cout<<"You will lose"<<"\n";
  }
  return 0;
}
